phoenix-hbase-compat-1.5.0/src/main/java/org/apache/phoenix/compat/hbase/coprocessor/CompatGlobalIndexChecker.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.phoenix.compat.hbase.coprocessor;
+
+import org.apache.hadoop.hbase.coprocessor.BaseRegionObserver;
+
+public class CompatGlobalIndexChecker extends BaseRegionObserver {
+    //HBase 1.5+ has preWALAppend already declared in BaseRegionObserver, so this class is only
```

Comment:
Super nit: multi-line comments throughout the pr have no space for the first line and 1 space for subsequent ones. Maybe keep it the same for all lines, unless this is some convention I'm unaware of, in which case ignore :D

Reply:
This particular file no longer exists, but I fixed some of these as I found them. The reason why is just that I usually don't type a space when I start a one-line comment, and when I find I need to make it multi-line the IDE fixes the spacing on subsequent lines but not the first one. :-)